kubernetes/config.properties

```diff
@@ -0,0 +1,8 @@
+inference_address=http://0.0.0.0:8080
+management_address=http://0.0.0.0:8081
+NUM_WORKERS=1
```

Comment:
@dhanainme We will probably need to describe this is a worker insider TorchServe and not the worker nodes on Kubernetes

Reply:
Done